lib/utils/ensData.ts

```diff
@@ -44,29 +42,26 @@ export interface ResolvedEnsData {
   owner?: string;
 }
 
-// ENS Domain Coins utils
-export enum DomainAddressesSupportedCryptocurrencies {
-  BTC = "BTC",
-  LTC = "LTC",
-  DOGE = "DOGE",
-  ETH = "ETH",
-  BNB = "BNB",
-}
-
 export const cryptocurrencyToCoinType = {
-  [DomainAddressesSupportedCryptocurrencies.BTC]: "0",
-  [DomainAddressesSupportedCryptocurrencies.LTC]: "2",
-  [DomainAddressesSupportedCryptocurrencies.DOGE]: "3",
-  [DomainAddressesSupportedCryptocurrencies.ETH]: "60",
-  [DomainAddressesSupportedCryptocurrencies.BNB]: "714",
+  BTC: "0",
```

Comment:
I don't like the way the redefine multiple times, the word BTC, LTC, etc. across this file.

We should rely to a single definition of these that would be reused across our app.

Can you please provide a solution that achieves this?

Reply:
I removed the enum to avoid double remapping(when you need to access an enum by its value, and then an object by its enum key), that can make the code very dirt, I'll try to find a more concise and sophisticated solution that can avoid double remapping and also make the coin names singular across the codebase. Thanks for the observation